features/_your_results/period_percent_table.feature

```diff
@@ -0,0 +1,9 @@
+Feature:
+  As a user who has just completed the WPCC tool, I should be able to see helpful information on how the yearly increments is done. 
+
+  Scenario: Display table of percentages for each period
+    Given I am on the YourDetailsPage
+    When I fill in my details
+    And I click the Next button
+    And I move on to the results page
+    Then I should see a table with a column heading for each period
```

Comment:
@UmairIft actually I am not confusing. I am asking if we should display the table with the percentage calculated OR with the period percent only.